README.md

```diff
@@ -15,7 +15,7 @@ Using middleman, you can run a development server which will live-build the API
 Blueprint website.
 
 ```shell
-$ rake serve
+$ bundle exec rake
```

Comment:
This change isn't needed as `rake serve` executes the underlying code in the bundle

https://github.com/apiaryio/apiblueprint.org/blob/d341f5652f92c1e5a84c113b75adce4b11e4ae17/Rakefile#L2-L4